.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+Exceptionally long log output should be posted as a gist (https://gist.github.com).
+
+### Steps to reproduce the behavior
+
+Please include the following if applicable:
+
+- [ ] Manual steps to reproduce issue
+- [ ] The output of `l0 admin debug`
```

Comment:
Sensitive material! How about we either take the `token` out of `l0 admin debug` or have this just be bullets for the api and cli versions